code/game/machinery/newscaster.dm

```diff
@@ -199,8 +200,12 @@ var/list/obj/machinery/newscaster/allCasters = list() //Global list that will co
 				dat+= {"<HR><BR><A href='?src=\ref[src];create_channel=1'>Create Feed Channel</A>
 				<BR><A href='?src=\ref[src];view=1'>View Feed Channels</A>
 				<BR><A href='?src=\ref[src];create_feed_story=1'>Submit new Feed story</A>
-				<BR><A href='?src=\ref[src];menu_paper=1'>Print newspaper</A>
-				<BR><A href='?src=\ref[src];refresh=1'>Re-scan User</A>
+				<BR><A href='?src=\ref[src];menu_paper=1'>Print newspaper</A>"}
+				if(silence == 0)
```

Comment:
In this case, there's no functional difference, but you should always use the not operator when checking something for falsity like this. It becomes more important when something can be 0 *or* null.